eq-author-api/utils/html.js

```diff
@@ -1,7 +1,7 @@
 const cheerio = require("cheerio");
-const { isNull, toString } = require("lodash");
+const { isNull, toString, trim } = require("lodash");
 
-const isHtml = value => !isNull(cheerio(value).html());
+const isHtml = value => !isNull(cheerio(trim(value)).html());
```

Comment:
Why do we need to trim?

Reply:
This is to make sure white space is accounted for, incase there is white space either side of the tags for some reason

Reply:
Did we find that anywhere?